### PR TITLE
Release 0.6.18

### PR DESCRIPTION
**Pull request by oooookk7**
AdapterJS: `0.14.1`
socket.io-client: `1.4.8`

### CODE REVIEW

File: demo/README.md [hunk 1]
```diff
@@ -117,4 +117,22 @@ In this demo, you can find an example of how to utilize the privileged key conce
 `getPeersStateChange`, `introduceStateChange`
 
 ##### How to use
-First, open the index page of the privileged demo. Each button there when being clicked will open another browser tab for the corresponding type of user, with corresponding functionalities available. For example privileged users have getPeers() and introduce() options while the rest do not. To introduce two peers, enter the peer ID of each peer to the textbox then hit "Introduce".
\ No newline at end of file
+First, open the index page of the privileged demo. Each button there when being clicked will open another browser tab for the corresponding type of user, with corresponding functionalities available. For example privileged users have getPeers() and introduce() options while the rest do not. To introduce two peers, enter the peer ID of each peer to the textbox then hit "Introduce".
+
+### demo/data-streaming
+In this demo, you can find an example of how you can share video (mp4) with peers with Skylink.
```
Comment (ncurrier) [inline demo/README.md]: with the Temasys Web SDK

File: demo/README.md [hunk 2]
```diff
@@ -117,4 +117,22 @@ In this demo, you can find an example of how to utilize the privileged key conce
...
+
+##### Functionalities covered
+`init()`, `joinRoom()`, `on()`, `streamData()`, `sendP2PMessage()`
+
+##### Events used
+`incomingDataStream`, `incomingMessage`, `peerLeft`, `dataChannelState`
+
+### demo/screenshare-and-camera
+In this demo, you can find an example of how you can share both camera and screenshare to peers with Skylink.
```
Comment (ncurrier) [inline demo/README.md]: with the Temasys Web SDK

File: README.md [hunk 3]
```diff
@@ -7,13 +7,18 @@ We've gone to great length to make this library work in as many browsers as poss
...
-| **Video Call**     | Yes        | Yes         | Yes       | Yes with [Temasys Plugin](http://skylink.io/plugin/)  | Yes [Temasys Plugin](http://skylink.io/plugin/)  |
-| **Audio Call**     | Yes        | Yes         | Yes       | Yes with [Temasys Plugin](http://skylink.io/plugin/) | Yes with [Temasys Plugin](http://skylink.io/plugin/)  |
-| **File Transfers** | Yes        | Yes         | Yes       | Yes with [Temasys Plugin](http://skylink.io/plugin/)  | Yes with [Temasys Plugin](http://skylink.io/plugin/)  |
-| **Chat Messaging** | Yes        | Yes         | Yes       | Yes with [Temasys Plugin](http://skylink.io/plugin/)  | Yes with [Temasys Plugin](http://skylink.io/plugin/)  |
+
+> ##### Browsers in Beta (Edge/Bowser):
+> _Do note that MCU connections, file transfers and P2P messages does not work in Edge browsers due to its current supports and beta phase. Earlier Edge browser versions might require you to toggle the H264 experimental flag to enable video connections from Edge to Chrome/Firefox browsers. Bowser browser is only officially supported by iOS versions `9.x` and below,_
```
Comment (ncurrier) [inline README.md]: Please note that MCU functionality, data channel file transfer, and P2P messaging are not supported by the Temasys Web SDK in Edge browser due to current compatibility limitations. Older versions of Edge may require you to enable experimental support for H.264 to interop with Chrome and Firefox browsers. Edge support should be considered experimental in this release.

File: bower.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "skylinkjs",
   "description": "WebRTC real-time video conversation library",
-  "version": "0.6.17",
+  "version": "0.6.18",
   "homepage": "https://skylink.io",
```
Comment (ncurrier) [inline bower.json:5]: https://temasys.io

File: demo/README.md [hunk 5]
```diff
@@ -117,4 +117,22 @@ In this demo, you can find an example of how to utilize the privileged key conce
 `getPeersStateChange`, `introduceStateChange`
 
 ##### How to use
-First, open the index page of the privileged demo. Each button there when being clicked will open another browser tab for the corresponding type of user, with corresponding functionalities available. For example privileged users have getPeers() and introduce() options while the rest do not. To introduce two peers, enter the peer ID of each peer to the textbox then hit "Introduce".
\ No newline at end of file
+First, open the index page of the privileged demo. Each button there when being clicked will open another browser tab for the corresponding type of user, with corresponding functionalities available. For example privileged users have getPeers() and introduce() options while the rest do not. To introduce two peers, enter the peer ID of each peer to the textbox then hit "Introduce".
```
Comment (ncurrier) [inline demo/README.md]: First, open the index page of the privileged demo. Each button there when clicked will open another browser tab for the corresponding type of user, with corresponding functionalities available. For example, privileged users have getPeers() and introduce() options while the rest do not. To introduce two peers, enter the peer ID of each peer to the text box then hit "Introduce".

File: demo/app/index.html [hunk 6]
```diff
@@ -229,7 +229,8 @@
     </div>
 	</footer>
   <script src="../../publish/skylink.complete.js"  type="text/javascript"></script>
-	<script src="../config.js"   type="text/javascript"></script>
+	<script src="../edge-attachmediastream.js"   type="text/javascript"></script>
```
Comment (ncurrier) [inline demo/app/index.html:232]: what the extra indent?

Comment (oooookk7) [reply demo/app/index.html:232]: screen shot 2017-03-03 at 5 43 53 pm

Hmm this is strange because on my sublime editor it appears no extra indentation.

**Review by ncurrier — APPROVED** [review]
ok for release

**Pull request merged**